tests/regression/87-once/11-combination.c

```diff
@@ -0,0 +1,45 @@
+// PARAM: --set ana.activated[+] pthreadOnce --set ana.activated[+] threadJoins --set ana.path_sens[+] threadflag --set ana.base.privatization mutex-meet-tid
+// pthread_once object is passed as a pointer (and it may change)
+#include <pthread.h>
+#include <stdio.h>
+
+int* g;
+pthread_once_t once = PTHREAD_ONCE_INIT;
+pthread_mutex_t mtx;
+
+void init0() {
+  g = malloc(sizeof(int));
+}
+
+
+void* thread(void* arg) {
+  pthread_once(&once, init0);
+
+  pthread_mutex_lock(&mtx);
+  if(g == NULL) {
+    // Will not happen!
+    // Also showing that g is not NULL here requires "Extended Support for pthread once" as explained in appendix C.
```

Comment:
Why not `__goblint_check(0); // NOWARN (unreachable)` then? Or is this extended support not implemented?